gender_analysis/common.py

```diff
@@ -111,124 +111,68 @@
-        ...                   'texts', 'austen_persuasion.txt')
-        >>> novel_text = f.load_file(novel_path)
-        >>> type(novel_text), len(novel_text)
-        (<class 'str'>, 486253)
-
-        csv files are returned as a list of strings, which can be
-        further processed with Python's csv module
-
-        >>> corpus_metadata_path = Path('testing', 'corpora', 'sample_novels',
-        ...                             'sample_novels.csv')
-        >>> corpus_metadata = f.load_file(corpus_metadata_path)
-        >>> type(corpus_metadata)
-        <class 'list'>
-
-        file_path can be a string or Path object
-
-        >>> import os
-        >>> novel_path_str = os.sep.join(['testing', 'corpora', 'sample_novels',
-        ...                               'texts', 'austen_persuasion.txt'])
-        >>> novel_text_str = f.load_file(novel_path_str)
-        >>> novel_text == novel_text_str
-        True
-        >>> novel_path2 = Path(r"testing/corpora/test_books_30/20-0.txt")
-        >>> paradise_lost = f.load_file(novel_path2)
-        >>> paradise_lost[1:61]
-        'The Project Gutenberg EBook of Paradise Lost, by John Milton'
-
-        Returns a str (txt file) or list of strs (csv file)
-        """
-        # if the file_path is a string, turn to Path object.
-        if isinstance(file_path, str):
-            file_path = Path(file_path)
-
-        # make sure that we only try to load supported file types
-        supported_file_types = {'.csv', '.txt'}
-        current_file_type = file_path.parts[-1][file_path.parts[-1].rfind('.'):]
-        if current_file_type not in supported_file_types:
-            err = "The FileLoaderMixin currently supports "
-            err += "{supported_file_types} but not {current_file_type}."
-            raise ValueError(err)
-
-        # check if we are working locally and in the correct directory
-        # __file__ is only available if executed from a file but
-        # not from an ipython shell or notebook
-        # in those cases, the file has to be loaded remotely from github.
+    if isinstance(file_path, str):
+        file_path = Path(file_path)
+
+    if file_type != '.csv':
+        raise Exception(
+            'Cannot load if current file type is not .csv'
+        )
+    else:
+        file = open(file_path, encoding='utf-8')
+        result = file.readlines()
+
+    file.close()
+    return result
+
+def load_txt_to_string(file_path):
+    """
+    Loads a txt file
+
+    >>> from pathlib import Path
+    >>> from gender_analysis import common
+    >>> novel_path = Path(common.BASE_PATH, 'testing', 'corpora', 'sample_novels', 'texts', 'austen_persuasion.txt')
+    >>> novel_text = load_txt_to_string(novel_path)
+    >>> type(novel_text), len(novel_text)
+    (<class 'str'>, 486253)
+
+    :param file_path: can be a string or Path object
+    :return: the text as a string type
+    """
+    file_type = PurePosixPath(file_path).suffix
```

Comment:
One more tiny nitpick: here, you're initing a new PurePosixPath, and taking its suffix to get the file type -- but one line later, we're going to init a new Path object anyway. So if you just take the suffix of that, we save this extra object init and allocation.

(and I'm pretty sure Path has suffix, so you don't need to cast it to PurePosixPath)